src/main/java/org/openjdk/jextract/JextractTool.java

```diff
@@ -532,6 +550,31 @@ private int run(String[] args) {
                 SUCCESS;
     }
 
+    private Integer parseLibraries(String optionString, OptionSet optionSet, boolean useSystemLoadLibrary, Options.Builder builder) {
```

Comment:
I'd suggest to either use `int` and then `0` for a successful execution, or something like `OptionalInt` if you want to model optionality more explicitly.